### PR TITLE
Fix/toolbar title restoration

### LINKED ISSUE #204: Restore toolbar title after the activity was destroyed.
Toolbar title is not restored after the activity was destroyed.

ezgif-2124298054

### PR DESCRIPTION
**Pull request by ken5scal**
Fixed #204 

What I did was simply add onSaveInstanceState(Bundle outstate) that stores Title of Main Activity, and then, retrieves it on onCreate(Bundle savedInstanceState).

I tried to come up with more smart method, but couldn't come up with an idea.
Any constructive idea is welcomed.

### CODE REVIEW

File: app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java [hunk 1]
```diff
@@ -85,7 +95,7 @@ protected void onDestroy() {
     private void initView() {
         setSupportActionBar(binding.toolbar);
         ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this,
-                binding.drawer, binding.toolbar, R.string.open, R.string.close);
+                                                                        binding.drawer, binding.toolbar, R.string.open, R.string.close);
```
Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java]: Is this change needed?

**Comment by konifar** [comment]
Thanks! Great work

File: app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java (hunk 1 above)
Comment (ken5scal) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java]: Sorry, I have to work on my Formatting Seriously...and of course need to look at diff. 
I fixed my Code Style, so it wont happen next time.

Comment (konifar) [inline app/src/main/java/io/github/droidkaigi/confsched/activity/MainActivity.java]: Don't worry :)

**Comment by konifar** [comment]
LGTM

**Pull request merged**